### DISCUSSION TITLE
Add localization

### DISCUSSION THREAD

**Discussion by dreamy32** [spaciousejar/learn-git-with-me#4] [Ideas]
I will add localization to french to begin with :)

**Comment by spaciousejar** [comment]
> I will add localization to french to begin with :)

Great idea! Adding French localization will certainly make the project more accessible. If you need any help or have questions along the way, feel free to ask!

**Reply by dreamy32** [reply]
Will do!

**Comment by dreamy32** [comment]
@spaciousejar Do you have suggestions or comments about the localization implementation before I start this weekend?

**Reply by spaciousejar** [reply]
"Hi @dreamy32, Before beginning the localization this weekend, please consider the cultural and language differences of the target audience. Could you thoroughly test the localized version before finalizing the implementation? Best of luck!"

**Reply by dreamy32** [reply]
How do you recommend organizing localization?  
1. `en/docs` or 2. `docs/en`?

I initially tried placing the locales at the root (option 1) with the i18n configuration in the Next.js config, but it didn’t function as expected. However, placing the locales within content subfolders (option 2) works smoothly, and I find it a more intuitive structure.  

If you have a preference or see potential issues with the second approach, please let me know so I can adjust accordingly.

**Reply by spaciousejar** [reply]
The first approach of placing locales at the root didn’t work, I suggest you use the second approach instead. Be sure to update your Next.js configuration to match the new directory structure.

If you face any specific issues with this setup or need more help, just let me know!
